src/components/DataTable.vue

```diff
@@ -1,75 +1,105 @@
 <template>
   <div class="container">
-    <table class="data-table display">
+    <table class="data-table">
       <thead>
         <tr>
           <th>Sr#</th>
           <th>Author</th>
           <th>Title</th>
+          <th>Actions</th>
         </tr>
       </thead>
       <tbody>
-        <tr v-bind:key="tableItem + index" v-for="(tableItem, index) in tableData">
+        <tr
+          v-bind:key="tableItem + index"
+          v-for="(tableItem, index) in tableData"
+        >
           <td>{{ index + 1 }}</td>
           <td>{{ tableItem.data.author }}</td>
           <td>{{ tableItem.data.title }}</td>
+          <td>
+            <button class="btn btn-primary" @click="showRowDetail(tableItem.data)">
+              Detail
+            </button>
+          </td>
         </tr>
       </tbody>
     </table>
+    <PostList v-on:deleteRow="deleteRow($event)" :post-details="postDetails" />
   </div>
 </template>
 
 <script>
 import axios from "axios";
 import $ from "jquery";
+import PostList from "./PostList.vue";
 
 export default {
   name: "Data-Table",
+  components: {
+    PostList,
+  },
   data() {
     return {
-      tableData: []
+      tableData: [],
+      showDetail: false,
+      postDetails: [],
     };
   },
 
-  methods:{
-    
-    searchDelay:()=>{
+  methods: {
+    showRowDetail(tableItem) {
+      this.postDetails.push(tableItem)
+      console.log(this.postDetails)
 
-    let myDataTable;
-    $(document).ready(function() {
-      myDataTable = $(".data-table")
-        .dataTable()
-        .api({
-          data: this.tableData,
-          columns: [{ title: "sr#" }, { title: "Author" }, { title: "Title" }],
-        
-      
-        });
+    },
 
-      const inputBox = $("input").attr("aria-name", "DataTables_Table_0");
+    deleteRow:function(data){
```

Comment:
Please follow same syntax for function definition. `func1 () {}` or `func1: function () {}`